apollo-federation/src/sources/connect/json_selection/methods/future/and.rs

```diff
@@ -48,7 +48,11 @@ fn and_method(
             None,
             vec![ApplyToError::new(
                 format!("Method ->{} requires arguments", method_name.as_ref()),
-                input_path.to_vec(),
+                input_path
+                    .to_vec()
+                    .into_iter()
+                    .map(|safe_json| safe_json.into())
+                    .collect(),
```

Comment:
I don't think we use `InputPath::to_vec` for anything other than this, so we might be able to change its return type to avoid most of the noisy changes in this PR (i.e., lines like this could stay `input_path.to_vec(),`).

Reply:
a68308e12